test/new_relic/agent/log_event_aggregator_test.rb

```diff
@@ -343,14 +376,24 @@ def test_high_security_mode
           'Supportability/Logging/Ruby/LogStasher/enabled' => {:call_count => 1},
           'Supportability/Logging/Metrics/Ruby/enabled' => {:call_count => 1},
           'Supportability/Logging/Forwarding/Ruby/enabled' => {:call_count => 1},
-          'Supportability/Logging/LocalDecorating/Ruby/disabled' => {:call_count => 1}
+          'Supportability/Logging/LocalDecorating/Ruby/disabled' => {:call_count => 1},
+          'Supportability/Logging/Labels/Ruby/disabled' => {:call_count => 1}
         },
           :ignore_filter => %r{^Supportability/API/})
       end
     end
 
     def test_overall_disabled
-      with_config(LogEventAggregator::OVERALL_ENABLED_KEY => false) do
+      # set the overall enabled key to false
+      # put all other configs as true
+      with_config(
+        LogEventAggregator::OVERALL_ENABLED_KEY => false,
+        :'instrumentation.logger' => 'auto',
+        LogEventAggregator::METRICS_ENABLED_KEY => true,
+        LogEventAggregator::FORWARDING_ENABLED_KEY => true,
+        LogEventAggregator::DECORATING_ENABLED_KEY => true,
+        LogEventAggregator::LABELS_ENABLED_KEY => true
+      ) do
```

Comment:
This is unrelated to the labels change. Happy to put it in a separate PR. Essentially, I wanted to make sure that the overall enabled shutoff really flipped the values for all related configs. Since labels are disabled by default, the disabled assertion below felt like a potential false positive.